gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -236,7 +251,8 @@ public void When_MappingMedicationRequestWithAuthoredOnValidDate_Expect_Authored
         var expectedAuthoredOn = DateFormatUtil.parseToDateTimeType("20100115");
 
         when(medicationRequestPlanMapper.mapToPlanMedicationRequest(any(), any(), any(), any())).thenReturn(new MedicationRequest());
-        when(medicationRequestOrderMapper.mapToOrderMedicationRequest(any(), any(), any(), any())).thenReturn(new MedicationRequest());
+        when(medicationRequestOrderMapper.mapToOrderMedicationRequest(any(), any(), any(), any(), any()))
```

Comment:
Not directly related to your changes as you didn't create the tests but just to mention as it looks like tests When_MappingMedicationRequestWithAuthoredOnValidDateInExtractOnly_Expect_AuthoredOnToUseAvailabilityTimeInStatement, When_MappingMedicationRequestWithAuthoredOnInExtractAndComposition_Expect_AuthoredOnToUseAvailabilityTimeInStatement, When_MappingMedicationRequestWithAuthoredOnValidDate_Expect_AuthoredOnToUseMedicationAvailabilityTime, When_MappingMedicationRequestWithAvailabilityTimeNotInMedicationStatement_Expect_AuthoredOnMapped, When_MappingMedicationRequestWithAvailabilityTimeInMedicationStatement_Expect_UseThatAvailabilityTime can be converted to Parameterized Tests which should reduce a lot of code in this Test class.

Reply:
Great spot! I'm going to take note of this, but in terms of scope I'll say it falls outside of `NIAD-3128` 😄